build-support/travis/before_install.mustache

```diff
@@ -0,0 +1,10 @@
+    - PATH="/usr/lib/jvm/java-8-openjdk-amd64/jre/bin":$PATH
```

Comment:
Since the mustache exists primarily to drive the sharding, this is convenient, but I think this can be done with anchors. This all could be anchored as a single command string instead of a list and then inserted as a single list item allowing other list items to be added if needed.

An example of this with env vars instead of commands is here:
declare:
https://github.com/pantsbuild/pex/blob/3f14b0416105770ca3443a362e8c0b1827365807/.travis.yml#L3-L5

use:
https://github.com/pantsbuild/pex/blob/3f14b0416105770ca3443a362e8c0b1827365807/.travis.yml#L43

Reply:
Not super strongly.